File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 7/13/2026, 10:16:17 AM
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        c=0
        k=len(nums)
        for i in range(k):
              if nums[i]!=val:
                    nums[c]=nums[i]
                    c+=1
      
        return c
        
        

File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 7/13/2026, 10:16:11 AM
class Solution {
    public int removeElement(int[] nums, int val) {
        
        int c=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]!=val){
                nums[c]=nums[i];
                c++;}}
        return c;

    }
}